src/main/resources/db/migration/V5__imrt_ddl.sql

```diff
@@ -0,0 +1,5 @@
+/******************************************************************************
+* Add associated stimulus id to the item table
+******************************************************************************/
+
+ALTER TABLE item ADD COLUMN associated_stimulus_id character varying(40);
```

Comment:
Note that I'm setting a max width.  Max width of values have been defined and approved within the item data dictionary.

Reply:
Also, this is called stimulusId within the item.json.  I can rename it to that if we feel that is a better idea.  I wasn't sure if that would be confusing to us long term.